### DISCUSSION TITLE
ClouDNS

### DISCUSSION THREAD

**Discussion by nice42q** [team-exor/generic-seeder#6] [Ideas]
Can ClouDNS be added to the project?
The API documentation https://www.cloudns.net/wiki/article/42/

Best regards

**Comment by joeuhren** [comment]
I'm sure anything is possible, although I'm not familiar with ClouDNS. I'm curious about which ClouDNS features you would be interested to see working with the seeder app? Is it just the fact that they have an api so you can do reporting on historical DNS requests or something to that effect?

Also, maybe I'm misunderstanding what kind of changes would be required to get ClouDNS working with the seeder, but could you not just configure your ClouDNS account with the DNS entries required for the seeder and use those without any code changes on this end?

**Comment by joeuhren** [comment]
No reply for years. Closing for now. Feel free to reopen this discussion if you still feel that ClouDNS could help achieve something better or different than can be done without.
